moncli/types.py

```diff
@@ -103,3 +103,38 @@ class TextType(MondayType):
     native_type = str
     allow_casts = (int, float)
     null_value = ""
+
+
+class DateType(MondayType):
+
+    native_type = datetime
+    allow_casts = (str, int)
+    null_value = {}
+    has_time = False
+
+    def __init__(self, id: str = None, title: str = None, has_time: bool = False, *args, **kwargs):
+        self.id = id
+        self.title = title
+        self.has_time = has_time
+        super().__init__(self.id, self.title, *args, **kwargs)
+
+    def _cast(self, value):
+        try:
+            if isinstance(value, int):
+                return datetime.fromtimestamp(value)
+            if isinstance(value, str):
+                date = datetime.strptime(value.split(' ', 1)[0], DATE_FORMAT)
```

Comment:
I forgot some requirements.  We need to verify that the passed in string has a valid format.  Otherwise we throw a ConversionError.  Please see requirements docs with the update.